code/game/machinery/mechlaunchpad.dm

```diff
@@ -61,6 +61,7 @@
 	style = STYLE_SEETHROUGH
 	explosionSize = list(0,0,0,0)
 	reversing = TRUE
+	reverseOptionList = list("Mobs"=FALSE,"Objects"=FALSE,"Anchored"=TRUE,"Underfloor"=FALSE,"Wallmounted"=FALSE,"Floors"=FALSE,"Walls"=FALSE) //Allow the pod to pick up anchored objects
```

Comment:
Some alternatives to this, just food for thought:
1. change this to be in an overridden Initialize() like so - 
`reverseOptionList["Anchored"]=TRUE`
might be cleaner
2. Just change mechpod's insertion_allowed to not call the super function
ie remove the `. = ..()`

Reply:
That was my original approach and was told by @Timberpoes that not calling the super there would be rejected as bad practice. I think that's much cleaner than relying on somebody to not change drop pods insertion_allowed proc yet again and break mech pods yet again.

Reply:
I'd prefer not to have the parent proc bypassed. It exists for a reason and skipping it is poor practice without a good reason.

"This thing doesn't have the correct configuration options set and I don't like the way you set the configuration options for this thing" is not one of those good reasons.

I'm happy with doing it the way msgerbs chose for one reason - We're explicitly defining the options for our reverse pickup and saying "No, none of these please. Just the anchored".

Reply:
In that case, it would be more appropriate to add a "mecha" field to the original reverseOptionList, and not even override the insertion_allowed proc from the first place - no?

Reply:
Stop being so smart @MrDoomBringer! I'll look at doing that, that is a vastly better idea.

Reply:
Did this aswell, seems to work very nicely. Also updated the admin window for pods (holy crap is that a gorgeous ui)